loki/logpipeline-custom.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: LogPipeline
+metadata:
+  name: logpipeline-sample
+spec:
+  input:
+    application:
+      namespaces:
+        system: true
+  output:
+    custom: |
+      name   loki
+      host   {release-name}-headless.{namespace}.svc.cluster.local
+      port   3100
+      labels job=fluentbit
+      auto_kubernetes_labels on
```

Comment:
The auto labellling option might be heavy.. the custom label map would help. On the other hand, the user can optimize it on his own.. maybe giving at least a hint that you can easily run into cardinality problems (there are nice articles) and providing an example how you would define your own map? As it is based on the new plugin, you finally can define it as a one-liner. @chrkl what do you think?

Reply:
The auto labelling is not really usable. I get these labels on a Kyma cluster:

app
app_kubernetes_io_component
app_kubernetes_io_instance
app_kubernetes_io_managed_by
app_kubernetes_io_name
app_kubernetes_io_part_of
app_kubernetes_io_version
chart
component
control_plane
controller_revision_hash
eventing_kyma_project_io_backend
gardener_cloud_role
helm_sh_chart
heritage
install_operator_istio_io_owning_resource
istio
istio_io_rev
job
k8s_app
kyma_project_io_component
kyma_project_io_dashboard
name
nats_cluster
networking_gardener_cloud_from_seed
networking_gardener_cloud_to_apiserver
networking_gardener_cloud_to_dns
networking_gardener_cloud_to_public_networks
operator_istio_io_component
origin
pod_template_generation
pod_template_hash
pool
release
resources_gardener_cloud_managed_by
role
security_istio_io_tlsMode
service_istio_io_canonical_name
service_istio_io_canonical_revision
shoot_gardener_cloud_no_cleanup
sidecar_istio_io_inject
statefulset_kubernetes_io_pod_name
version

Too much clutter and important labels like namespace or container are missing.

Reply:
I think that might not even be a cardinality problem since the labels do not contain any dynamic information from the log records, but still not nice to use.